packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-output.vue

```diff
@@ -30,7 +30,11 @@
 		</AccordionTab>
 		<AccordionTab>
 			<template #header>Parameters<i class="pi pi-info-circle" /></template>
-			<vega-chart :visualization-spec="parameterCharts" :are-embed-actions-visible="false" />
+			<vega-chart
+				:visualization-spec="parameterCharts"
+				:are-embed-actions-visible="false"
+				@chart-click="onParameterChartClick"
```

Comment:
Can you explain a bit how the visual encodings work with respect to click on the parameters' rug plot? I played around with it and it is confusing what had been selected.